### ISSUE TITLE
0.5 Sparse Tensor Add Error

### ISSUE THREAD

**Issue by rancheng** [NVIDIA/MinkowskiEngine#257]
test script from `sparse_tensor_basic.py`:
```python
# input data
coords0 = torch.tensor([[0, 0, 2],
        [0, 1, 1],
        [0, 1, 2],
        [0, 1, 3],
        [0, 2, 2]], dtype=torch.int32)
coords1 = torch.tensor([[0, 0, 0],
        [0, 1, 1],
        [0, 2, 2]], dtype=torch.int32)
feats0 = torch.tensor([[2.1000],
        [1.0000],
        [1.4000],
        [3.0000],
        [4.0000]])
feats1 = torch.tensor([[1.],
        [2.],
        [3.]])

A = ME.SparseTensor(coordinates=coords0, features=feats0)
# B = ME.SparseTensor(coordinates=coords1, features=feats1)

B = ME.SparseTensor(
        coordinates=coords1,
        features=feats1,
        coordinate_manager=A.coordinate_manager,
        coordinate_map_key=A.coordinate_map_key
    )

# /tmp/MinkowskiEngine/src/coordinate_map_manager.cpp:315, (true) CoordinateMapKey collision detected: (1 1, ) generating new string id.

C = A + B
```

error message:
```sh
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/IPython/core/interactiveshell.py", line 3343, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-8-49c8b7dac430>", line 1, in <module>
    C = A + B
  File "/usr/local/lib/python3.6/dist-packages/MinkowskiEngine/MinkowskiTensor.py", line 713, in __add__
    return self._binary_functor(other, lambda x, y: x + y)
  File "/usr/local/lib/python3.6/dist-packages/MinkowskiEngine/MinkowskiTensor.py", line 686, in _binary_functor
    ins, outs = self._manager.get_union_map(
AttributeError: 'CoordinateManager' object has no attribute 'get_union_map'
```

Did I miss anything in creating `B`?

**Comment by chrockey** [comment]
I heard that union_map is not implemented yet in MinkowskiEnginev0.5.

**Comment by chrischoy** [comment]
Implemented on  fa88fa2434
